site/content/docs/alerting-and-retries/retries.md

```diff
@@ -7,61 +7,107 @@ menu:
 cli: true
 ---
 
-Sometimes the internet is flaky, or your app is just having a hiccup not worth pinging your on-call team about. Retries 
-are your first line of defense against these types of false positives, leading to alert fatigue.
+Sometimes the internet is flaky, or your app is just having a hiccup not worth pinging your on-call team about. Retries are your first line of defense against these types of false positives, leading to alert fatigue.
+
+Whether you need to retry a check or a test session, there are different retries available.
 
 {{<warning>}}
-The "double check" option is deprecated and will be removed in the future. Please use the retries instead.
+Use **Retries** instead of the deprecated "double check" option.
 {{</warning>}}
 
+## Check retry strategies
 
-## Retry strategies
-
-Each check (except for Heartbeat checks) and each check group has a **Retries** section that allows you to configure how many times
-and according to which strategy a check should be retried.
+All check types and check groups, except for Heartbeat checks, have a **Retries & Alerting** section under **Edit > Check settings**, where you can define the number of retries and which retry strategy your check or check group uses.
```

Comment:
**Edit > Check settings** is the path for browser & multistep checks, for API checks its **Edit** - I got confused by this for a second, not sure if its worth listing separately, wdyt?

Reply:
we should improve this on the product for sure :)

In the meantime, I've just pushed an update for those that might get confused for a second 😅 (Went with the most straight-forward path to get there :) what do you think?

Reply:
Ah yes, I was referring to separating it in the docs - Changing the app is for another time 😄
I think its very clear now! 🙂